File: code/chemical-main/frontend/src/api/car.ts
import request from '@/utils/request'
import type { ApiResult } from '@/types/api'

export interface CarRecord {
  id?: number
  carId: number
  warning: 0 | 1
  x: number
  y: number
  gasType?: string | null
}

export const reqCarList = () =>
  request.get<null, ApiResult<CarRecord[]>>('/car/getAllCars')

export const reqSetCarWarning = (carId: number) =>
  request.post<{ carId: number }, ApiResult<string>>('/car/setWarning', { carId })

export const reqResetCarStatus = (carId: number) =>
  request.post<{ carId: number }, ApiResult<string>>('/car/resetStatus', { carId })

File: code/chemical-main/backend/src/main/java/com/at/pojo/Car.java
package com.at.pojo;

import lombok.Data;

@Data
public class Car {
    private Integer id;         // 数据库主键
    private Integer carId;      // 小车ID
    private Integer warning;    // 0=正常 1=预警
    private Integer x;          // X坐标
    private Integer y;          // Y坐标
    private String gasType;     // 检测气体类型
}